test/e2e/http_header_buffer_test.go

```diff
@@ -115,7 +115,7 @@ func TestHTTPHeaderBufferSize(t *testing.T) {
 		"-H",
 		headerString,
 	}
-
+	extraCurlArgs = append(extraCurlArgs, "--resolve", echoRoute.Spec.Host+":80:"+service.Spec.ClusterIP)
```

Comment:
Not 100% sure how this is related, not that I care too much. But I see you are manually resolving the IP. Why exactly? Is this related to this PR?

Reply:
So the `--resolve` option was added by default in the func `buildCurlPod()` and was being mapped to `address` (specifically IP addresses). It **does not** allow CNAME to be provided as the destination address.

For our use-case I'm using `--resolve` to map a dummy domain to the target associated with the ingresscontroller (on AWS this is a CNAME). So I've moved all the usages of `--resolve` out of the function.

Reply:
Ah okay I see it now. Thanks for explaining.